tests/response/cas-success-service-response.xml

```diff
@@ -0,0 +1,17 @@
+<cas:serviceResponse xmlns:cas="http://www.yale.edu/tp/cas" xmlns:slate="https://example.org/slate">
+    <cas:authenticationSuccess>
+        <cas:user>jdoe</cas:user>
+        <cas:attributes>
+            <cas:authenticationDate>2025-11-07T22:00:24+02:00</cas:authenticationDate>
+            <cas:longTermAuthenticationRequestTokenUsed>true</cas:longTermAuthenticationRequestTokenUsed>
+            <cas:isFromNewLogin>true</cas:isFromNewLogin>
+            <cas:sn>Doe</cas:sn>
+            <cas:firstname>John</cas:firstname>
+            <cas:mail>jdoe@example.edu</cas:mail>
+            <cas:eduPersonPrincipalName>jdoe@example.edu</cas:eduPersonPrincipalName>
+        </cas:attributes>
+        <slate:person>12345</slate:person>
```

Comment:
I don't know much about CAS, but the schema doesn't seem to allow arbitrary elements here.
What's the idea behind this?